carbon-design-system/carbon-addons-iot-react
# feat(timeseriescard): expose the thresholds for x and y axis for time series chart

Closes #

**Summary**

- Carbon charts has added a thresholds prop and we're exposing this now in the TimeSeriesCard

**Change List (commits, features, bugs, etc)**

- fix(package.json): add helper builder react jsx to support builds, they started breaking
- fix(storybooksnapshots): mock the document and window for getComputedStyle
- feat(TimeSeriesCard): add thresholds prop

**Acceptance Test (how to verify the PR)**

- view the thresholds story of the TimeSeriesCard

**Regression Test (how to make sure this PR doesn't break old functionality)**

- Verify a few of the other TimeSeriesCard stories
